docs/user_guide/api/LudwigModel.md

```diff
@@ -487,6 +488,9 @@ Using a trained model, make predictions from the provided dataset.
   to use. Possible values are `'full'`, `'training'`, `'validation'`, `'test'`.
 - **batch_size** (int, default: 128): size of batch to use when making
   predictions.
+- **generation_config** (Dict, default: `None`) config for the generation of the
```

Comment:
I'd also just make a mention that this is only used if the model type is LLM, otherwise it is ignored. 

Then, I'd also add a link to the docs for generation config (it's in configuration > large language models) indicating that users can see how to configure this parameter in those docs so that users know what kind of parameters they can use. 

Wdyt?

Reply:
On it

Reply:
Hmm. This page should be generated automatically using `code_doc_autogen.py`, which runs every night. I wonder why this page wasn't automatically updated.

Reply:
Ah, here's the PR: https://github.com/ludwig-ai/ludwig-docs/pull/303. In general, if there's documentation we want to add to this page, we should update `api.py` directly.

Reply:
This is the update to Ludwig: https://github.com/ludwig-ai/ludwig/pull/3535